integ-test/src/test/java/org/opensearch/sql/ppl/WhereCommandIT.java

```diff
@@ -110,4 +110,14 @@ public void testRelevanceFunction() throws IOException {
                 TEST_INDEX_BANK));
```

Comment:
Should the test name here specify that it is testing match specifically as there will be additional relevance tests? Possibly testMatchRelevanceFunction()?